plugins/Jobs.cpp

```diff
@@ -1195,6 +1196,34 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
         return true;
     }
 
+    // Requeue a job for which a getJob(s) command could not complete.
+    else if (SIEquals(requestVerb, "RequeueJobs")) {
+        SINFO("Requeueing jobs with IDs: " << command.request["jobIDs"]);
+        list<int64_t> jobIDs = SParseIntegerList(command.request["jobIDs"]);
+
+        SQResult result;
+        string selectQuery = "SELECT jobID, retryAfter FROM jobs WHERE jobID IN(" + SQList(jobIDs)+ ");";
+        db.read(selectQuery, result);
+
+        for (auto& row : result.rows) {
+            int64_t jobID = stoll(row[0]);
+            bool retryAfter = !row[1].empty();
+            string updateQuery;
+            if (retryAfter) {
+                // Reset the nextRun time to now, we don't need to wait for the timeout.
+                updateQuery = "UPDATE jobs SET nextRun = DATETIME("+ SCURRENT_TIMESTAMP() + ") WHERE jobID = " + SQ(jobID) + ";";
```

Comment:
We do this so it remains in `RUNQUEUED` state instead of `RUNNING` right?

Reply:
`RUNQUEUED` jobs normally stay `RUNQUEUED` until they're finished. We could do nothing here and they'd eventually retry anyway. We do this so they can retry sooner. If they have an 8-hour timeout, and we do nothing in this case, they'll sit for 8 hours, but since we already know they're going to need to retry, we might as well let them retry right away. Like @mea36 says, we could also make them `QUEUED` again, it doesn't really have any different effect.